File: ruby/arcade/allLongestStrings.rb
def allLongestStrings(inputArray)
	c = 0
	inputArray.each do |i|
		c = i.size > c ? i.size : c
	end
	s = []
	inputArray.each do |i|
		s.push i if i.size == c
	end
	return s
end

File: Arcade/Intro/Java/AllLongestStrings.java
import java.util.Arrays;

public class AllLongestStrings {

	String[] allLongestStrings(String[] inputArray) {
	    int max = Stream.<String>of(inputArray)
		.mapToInt(n -> n.length())
		.max()
		.getAsInt();
		
	    return Stream.<String>of(inputArray)
		.filter(e -> e.length() == max)
		.toArray(String[]::new);
	}
}
